backend/utils/validateOrder.js

```
@@ -0,0 +1,46 @@
+import mongoose from "mongoose";
+
+export const isValidUserId = (userId) => mongoose.isValidObjectId(userId);
+
+export const isValidItems = (items) => {
+  if (!Array.isArray(items) || items.length === 0) return false;
+
+  for (let item of items) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'item' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)